explorer/ui/app/components/details/TabList.tsx

```diff
@@ -55,22 +59,18 @@ export const TabList = ({ xMsgDetails }) => {
           onClick={() => copyToClipboard(xMsgDetails?.to)}
         />
       </div>
-      {/* Offset */}
       <div className="flex mt-5 pb-2 border-b-[1px] border-subtle border-solid">
         <p className="w-[150px] sm:w-48 text-sm">Stream Offset</p>
         <p className="text-default">{parseInt(xMsgDetails?.offset)}</p>
       </div>
-      {/* Offset */}
       <div className="flex mt-5 pb-2 border-b-[1px] border-subtle border-solid">
         <p className="w-[150px] sm:w-48 text-sm">Gas Used</p>
-        <p className="text-default">{parseInt(xMsgDetails?.receipt?.gasUsed)}</p>
+        <p className="text-default">{xMsgDetails?.receipt ? parseInt(xMsgDetails?.receipt?.gasUsed) : '--'}</p>
```

Comment:
Don't show `NaN` if receipt doesn't exist. Instead show `--` as we do with other fields in this case.